src/data/full-quiz.js

```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "What is the data type of (1) and (1,) ?",
+    Answer: "<class 'int'>  <class 'tuple'>",
+    Distractor1: "<class 'tuple'> <class 'int'>",
+    Distractor2: "<class 'int'>  <class 'tuple'>",

```

Comment:
The distractors should not be the same as the correct answer.
The distractors will always be the wrong answer. 

Proposed fix: 
Distractor2: "<class 'dictionary'>  <class 'list'>",